guides/common/modules/con_performing-post-upgrade-tasks.adoc

```diff
@@ -4,3 +4,7 @@
 Some of the procedures in this section are optional.
 You can choose to perform only those procedures that are relevant to your installation.
 
+ifdef::Satellite[]
+If you use custom templates created from a cloned default template, create the custom template again after the upgrade.
+Sometimes the default templates are changed during the upgrade, this will ensure that the custom templates are compatible with the latest version.
```

Comment:
This sentence has a comma splice. Either break the sentence into 2 sentences or replace the comma with a semi-colon.

ISG says to avoid the future tense and ambiguous pronouns. Suggestion: "Sometimes the default templates are changed during the upgrade. Re-creating custom templates ensures that your templates are compatible with the latest version."